src/forms/RadioButtonGroup.jsx

```diff
@@ -37,9 +38,23 @@ export default class RadioButtonGroup extends PureComponent {
 	}
 
 	render() {
-		const { direction, children, name, className } = this.props;
+		const {
+			direction,
+			switchDirection,
+			children,
+			name,
+			className
+		} = this.props;
+
+		const classNames = cx(
```

Comment:
If we remove the new CSS in `_togglePill.scss`, we can get rid of the classname stuff